docs/bok/Activities/Level-2/Creating-An-OSPO.md

```diff
@@ -5,17 +5,130 @@ tags:
   - OSPO (Role)
   - Placeholder
   - Level 2 (OSMM)
-draft: true
```

Comment:
It still has many "- tbd. ... add more examples here.", "(leave this part to Rob to fill in)", etc - isn't it too early to remove this?

Reply:
I'm removing it so that it appears in the preview.  We'll need to complete it properly before it's merged